data/human/hails.txt

```diff
@@ -2195,6 +2195,360 @@ phrase "friendly civilian"
+		"ship fit for mining, like a Sunder"
+		"Syndicate Sunder"
+		"Sunder and two mining drones"
+		"Sunder mining vessel"
+		"good mining ship, like a Sunder"
+		"defensible ship like a Headhunter"
+		"multi-purpose ship like a Headhunter"
+		"defensible ship like an Argosy"
+		"versatile ship like an Argosy"
+		"carrier like an Aerie. You can use the fighters to mine for you"
+		"squadron of Furies with mining lasers"
+		"ship with a large cargo hold"
+		"good freighter that can fit mining laser turrets"
+		"ship that can defend itself from pirates"
+		"powerful escort ship that can scare off pirates"
+		"ship that can fit loads of mining lasers"
+		"Behemoth loaded with mining laser turrets"
+
+phrase "miner advice: other"
+	word
+		"you should stick to patrolled systems so the pirates don't blow you up"
+		"you should stick to patrolled systems so you won't get extorted by pirates"
+		"you should avoid patrolled systems as they don't tend to have rare minerals"
+		"you should learn to recognize what asteroids are worthless or expensive"
+		"keep in mind that mining in systems frequented by pirates may be risky, but they often have good minerals"
+		"consider bribing pirates to leave you alone in systems with valuable asteroids"
+
+phrase "friendly civilian"
+	word
+		"I'm trying to improve my piloting skills without risking my life too much, so I've been mining asteroids."
+		"Mining can be a good way to improve your piloting skills."
+		"Mining is a safe career option for new Sparrow pilots."
+		"Asteroid miners should be better appreciated by other captains. It takes a lot of effort to destroy asteroids and just as much effort to scoop them up into your hold."
+		"I'm sick and tired of other ships stealing the minerals I've mined before I can get to them."
+
+phrase "friendly civilian"
+	word
+		"I find that %"
+		"Often, %"
+		"While difficult, %"
+		"It looks difficult, but %"
+	word
+		"Mining"
+		"Harvesting"
+	word
+		" asteroids "
+	word
+		"can be "
+		"is "
+	word
+		"surprisingly "
+		"quite "
+		"very "
+	word
+		"profitable"
+		"fun"
+		"relaxing"
+		"worthwhile"
+		"meditative"
+	word
+		" and "
+	word
+		"profitable"
+		"fun"
+		"relaxing"
+		"worthwhile"
+		"meditative"
+	word
+		"."
+	replace
+		"%M" "m"
+		"%H" "h"
+		"profitable and profitable" "profitable"
+		"fun and fun" "fun"
+		"relaxing and relaxing" "relaxing"
+		"worthwhile and worthwhile" "worthwhile"
+		"meditative and meditative" "meditative"
+
+phrase "friendly civilian"
+	word
+		"It's becoming harder"
+		"It used to be much easier"
+		"I wish it was easier"
+		"Once it was quite easy"
+		"Once it was easier for captains"
+	word
+		" to work as "
+		" to be "
+	word
+		"an asteroid miner"
+		"a miner"
+		"an independent miner"
+	word
+		"."
+		"..."
+	word
+		"" 3
+		" Pirate extortion fees are becoming just another business expense."
+		" The pirate fleets get bolder and more aggressive every day, but the Navy only patrols in the Paradise Worlds."
```

Comment:
I feel like these don't line up well with the FW campaign reducing piracy.

Reply:
Just this one? I can remove it.
```suggestion
```